src/twisted/python/test/_deprecatetests.py.3only

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
```

Comment:
Will this file actually get included in the distribution?  Having a Python source file without a ".py" extension sounds like it will trip over all kinds of weirdness.

Does the CI system exercise installation of this code somewhere?